mobile/App.js

```diff
@@ -9,11 +9,29 @@
 import React from 'react';
 import {SafeAreaProvider} from 'react-native-safe-area-context';
 import HomeView from './src/views/HomeView';
+import ScanView from './src/views/ScanView';
+import RegistrationView from './src/views/RegistrationView';
+import {NavigationContainer} from '@react-navigation/native';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import ScreenOption from './src/components/share/ScreenOption';
+import {globalStyleSheet} from './src/components/share/Theme';
+
+const Tab = createBottomTabNavigator();
 
 const App = () => {
   return (
     <SafeAreaProvider>
-      <HomeView></HomeView>
+      <NavigationContainer>
+        <Tab.Navigator screenOptions={ScreenOption}>
```

Comment:
Could we make the icon and text in navigation smaller to have more space like in figma mockups? 
![image](https://user-images.githubusercontent.com/29143314/175830797-4c7a0394-9d01-4fcc-83fa-1c53a100ad9c.png)

Reply:
Fixed with 30b6b13a2e965c6f5bd3dd01fecc41e1bd38cc5d